drivers/clock_control/clock_control_ifx_fixed_clock.c

```diff
@@ -34,6 +38,56 @@ __WEAK void ifx_clock_startup_error(uint32_t error)
 	}
 }
 
+#define CY_CFG_SYSCLK_PLL_ERROR 3
+
+#if DT_NODE_HAS_STATUS_OKAY(DT_NODELABEL(dpll_lp0)) ||                                             \
+	DT_NODE_HAS_STATUS_OKAY(DT_NODELABEL(dpll_lp1))
+#define DPLL_LP0 SRSS_PLL_250M_0_PATH_NUM
+#define DPLL_LP1 SRSS_PLL_250M_1_PATH_NUM
+#endif
+
+#if DT_NODE_HAS_STATUS_OKAY(DT_NODELABEL(dpll_lp0)) ||                                             \
+	DT_NODE_HAS_STATUS_OKAY(DT_NODELABEL(dpll_lp1))
+void ifx_clk_dpll_lp_init(uint32_t dpll_lp, cy_stc_dpll_lp_config_t dpll_lp_config)
+{
+	cy_stc_pll_manual_config_t dpll_config = {
+		.lpPllCfg = &dpll_lp_config,
+	};
+
+#if !defined(CY_PDL_TZ_ENABLED)
+	if (Cy_SysClk_PllIsEnabled(dpll_lp)) {
+		return;
+	}
+#endif
+
+	Cy_SysClk_PllDisable(dpll_lp);
+	if (CY_SYSCLK_SUCCESS != Cy_SysClk_PllManualConfigure(dpll_lp, &dpll_config)) {
+		ifx_clock_startup_error(CY_CFG_SYSCLK_PLL_ERROR);
+	}
+
+#if (defined(CY_IP_MXS22SRSS_VERSION) && defined(CY_IP_MXS22SRSS_VERSION)) &&                      \
+	((CY_IP_MXS22SRSS_VERSION == 1) && (CY_IP_MXS22SRSS_VERSION_MINOR == 0))
+	/* The workaround for devices with MXS22SRSS block 1.0 */
+	uint32_t clkHfMask = Cy_SysClk_ClkHfGetMaskOnPath((cy_en_clkhf_in_sources_t)dpll_lp);
```

Comment:
zephyr is snake case not camel case (calling/using camel case hal functions/defines is fine but defining variables is not e.g. `clkHfMask`), fix in whole PR